device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-C64/qos.json.j2

```diff
@@ -164,8 +210,67 @@
             "6": "6",
             "7": "0"
         }
+{% endif %}
+    },
+{%- endmacro %}
+{%- macro generate_tc_to_queue_map() %}
+    "TC_TO_QUEUE_MAP": {
+{% if (('type' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['type'] == 'ToRRouter') and
+       ('subtype' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['subtype'] == 'DualToR')) %}
+        "AZURE_TUNNEL": {
+            "0": "0",
+            "1": "1",
+            "2": "2",
+            "3": "2",
+            "4": "6",
+            "5": "5",
+            "6": "6",
+            "7": "7",
+            "8": "1"
+        },
+        "AZURE": {
+            "0": "0",
+            "1": "1",
+            "2": "2",
+            "3": "3",
+            "4": "4",
+            "5": "5",
+            "6": "6",
+            "7": "7",
+            "8": "1"
+        }
```

Comment:
Please consider mapping `TC2`  and `TC6` to lossy queue. Even if there is no DSCP being mapped to `TC2` and `TC6`, mapping the never used value to a lossy queue should be better.

Reply:
As discussed offline, we have to map TC `2` and `6` to queue `2` and `6` respectively so that SAI will generate the correct settings to SDK/ASIC